### DISCUSSION TITLE
Plugin not loading in

### DISCUSSION THREAD

**Discussion by Alienprodigy** [rsjaffe/MIDI2LR#645] [Problems]
Version 4.3.1.11, tried updating to 5.3.0.2, however this doesn't seem to work.

When booting up Lightroom the plugin doesn't load in, the usually window doesn't pop up. When checking under plugin manager it states the plugin is installed and running.

When disabling, then enabling the app, then hitting reload plugin, the following errors show up:
Plug-in error log for plug-in at: C:\Program Files\Adobe\Adobe Lightroom Classic\MIDI2LR.lrplugin

**** Error 1

An error occurred while attempting to run one of the plug-in’s scripts.
?:0: attempt to call method 'send' (a nil value)

**** Error 2

An error occurred while attempting to shut down the plugin.
?:0: attempt to call method 'send' (a nil value)

additional info:
Log started: 31 Mar 2022 12:46:49pm

2022-03-31T12:46:49.533+02:00 CommandSet.cpp(71) MenuTrans.xml archive loaded from C:\Users\Michel\AppData\Roaming\MIDI2LR\MenuTrans.xml.
2022-03-31T12:46:49.538+02:00 Main.cpp(407) Exception Could not detect cereal root node - likely due to empty or invalid input
2022-03-31T12:46:49.539+02:00 Main.cpp(237) Exception Could not detect cereal root node - likely due to empty or invalid input
2022-03-31T12:46:49.539+02:00 Main.cpp(148) Terminate called, exception Could not detect cereal root node - likely due to empty or invalid input.

----------- LIGHTROOM -----------
Operating system Windows 10 - Business Edition (x64)
Version 11.2
Language en
Preferences path C:\Users\Michel\AppData\Roaming\Adobe\Lightroom\Preferences
Application data path C:\Users\Michel\AppData\Roaming\Adobe\Lightroom
----------- PLUGIN -----------
Version 4.3.1.11
Path C:\Users\Michel\AppData\Roaming\Adobe\Lightroom\Modules\MIDI2LR.lrplugin
MIDI2LR log path C:\Users\Michel\AppData\Roaming\MIDI2LR
MIDI2LR data path C:\Users\Michel\AppData\Roaming\MIDI2LR
----------- APP -----------
Application: System language en-GB.
Application: CPU AMD Ryzen 5 3600 6-Core Processor.
Application: Application version 4.3.1.11.

**Comment by rsjaffe** [comment]
Delete _C:\Users\Michel\AppData\Roaming\MIDI2LR\\_ and _C:\Users\Michel\AppData\Roaming\Adobe\Lightroom\Modules\MIDI2LR.lrplugin_ and reinstall.

**Comment by Alienprodigy** [comment]
Thank you, that has worked.
